shared/types/events.go

```diff
@@ -0,0 +1,51 @@
+package types
+
+import (
+	"google.golang.org/protobuf/types/known/anypb"
+)
+
+type SourceModule string
+type EventTopic string
+
+const (
+	// Core
+	CONSENSUS_MODULE SourceModule = "consensus"
+	P2P              SourceModule = "p2p"
+	persistence      SourceModule = "persistence"
```

Comment:
Let's get the format down, weird that some are all caps and others are all lower... Surely persistence isn't a private type

Reply:
@Olshansk?

Reply:
To be clear, my recommendation is:
```
CONSENSUS_MODULE SourceModule = "consensus"
P2P              SourceModule = "p2p"
PERSISTENCE      SourceModule = "persistence" // notice the name of the var
```

Reply:
Done. I also deleted the majority of topics in this file and we can add them back on a at-need basis.